Extensions/dnSpy.Debugger/dnSpy.Debugger/ToolWindows/CodeBreakpoints/CodeBreakpointVMComparer.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.Linq;
+using dnSpy.Contracts.Debugger.Code;
+using dnSpy.Contracts.Text.Classification;
+using dnSpy.Debugger.UI;
+
+namespace dnSpy.Debugger.ToolWindows.CodeBreakpoints {
+	sealed class CodeBreakpointVMComparer : FormatterObjectVMComparer<CodeBreakpointVM> {
+		public static readonly CodeBreakpointVMComparer Instance = new CodeBreakpointVMComparer(null, ListSortDirection.Ascending);
+
+		public CodeBreakpointVMComparer(string vmPropertyName, ListSortDirection direction) : base(vmPropertyName, direction) {}
+
+		protected override int doCompare(CodeBreakpointVM x, CodeBreakpointVM y) {
+			if (String.IsNullOrEmpty(this.VMPropertyName)) {
```

Comment:
If it's a one-line statement, the curly braces aren't needed and shouldn't be used.